extensions/interactions/LogicProof/static/js/student.js

```diff
@@ -994,12 +994,16 @@ var logicProofStudent = (function() {
   *          }
   */
   var buildProof = function(proofString, questionInstance) {
-    if (proofString.slice(-1) === '\n') {
-      proofString = proofString.slice(0, proofString.length - 1);
-    }
     var lineStrings = proofString.split('\n');
+    // Ignore blank lines at the end

```

Comment:
Also, what is your plan for handling blank lines in the beginning and in the middle?